src/backend/distributed/executor/multi_executor.c

```diff
@@ -660,10 +660,9 @@ ExecutePlanIntoDestReceiver(PlannedStmt *queryPlan, ParamListInfo params,
 	/* don't display the portal in pg_cursors, it is for internal use only */
 	portal->visible = false;
 
-	PortalDefineQuery(portal,
+	PortalDefineQuerySelectCompat(portal,
 					  NULL,
 					  "",
-					  "SELECT",
```

Comment:
c1a3000

Reply:
nit; any reason to not use that commit as a fixup ontop of this commit?
I don't have strong feelings around it, but given the plan to merge instead of squash it could help getting the blames on the work in `Use CMDTAG_SELECT_COMPAT` to this change, including the postgres commit that is linked to it.